Add `ticloud connect` doc (#12399)
pingcap/docs: tidb-cloud/tidb-cloud-release-notes.md

This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.

## February 17, 2023

**CLI changes**

> - Add new command [`ticloud connect`](/tidb-cloud/ticloud-connect.md)

Reviewer: ```suggestion
- Add a new command [`ticloud connect`](/tidb-cloud/ticloud-connect.md) for [TiDB Cloud CLI](/tidb-cloud/cli-reference.md).
```

>     `ticloud connect` is a command that allows you to connect to a TiDB Cloud cluster from your local machine. You can use `ticloud connect` to connect to a TiDB Cloud cluster and execute SQL statements.

Reviewer: ```suggestion
    `ticloud connect` is a command that allows you to connect to your TiDB Cloud cluster from your local machine without installing any SQL clients. After connecting to your TiDB Cloud cluster, you can execute SQL statements in the TiDB Cloud CLI.
```

>   `ticloud connect` is a command that allows you to connect to your TiDB Cloud cluster from your local machine without installing any SQL clients. After connecting to your TiDB Cloud cluster, you can execute SQL statements in the TiDB Cloud CLI.

Reviewer: ```suggestion
    `ticloud connect` is a command that allows you to connect to your TiDB Cloud cluster from your local machine without installing any SQL clients. After connecting to your TiDB Cloud cluster, you can execute SQL statements in the TiDB Cloud CLI.
```
